simulation/replication/run.sh

```diff
@@ -30,14 +30,16 @@ echo "Building test images"
 docker-compose -f docker/buildkite/docker-compose-local-replication-simulation.yml \
   build cadence-cluster0 cadence-cluster1 replication-simulator
 
-function check_test_failure {
-  echo "Checking test failure"
-  faillog="$(cat test.log | grep  'FAIL: TestReplicationSimulation111' -B 10)";
-  if [ -n "$faillog" ]; then
-    echo 'Test Failed!!!';
-    echo "$faillog";
-    echo "Check test.log file for more details";
-    exit 1;
+function check_test_failure()
+{
+  faillog=$(grep 'FAIL: TestReplicationSimulation' -B 10 test.log 2>/dev/null || true)
```

Comment:
do we check the exit code of the test? I would have thought that's a more standardized way to ensure the test is passing

Reply:
the script below does more things after the test is run. so exit code of last command is not a good indicator of whether the actual test have failed or not